website/docs/getting-started/building-an-image.md

```diff
@@ -0,0 +1,30 @@
+---
+sidebar_position: 10
+title: Building an image
+description: Building an image on your Container engine.
```

Comment:
curious why Container is with an uppercase

Reply:
mistake